.github/ISSUE_TEMPLATE/qa_issue_template.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Manual QA
+about: Use this template for running manual QA issues.
+title: "[DATE]: [FEATURE NAME]"
+labels: qa, review
+---
+# Manual QA for # 
+
+### Check list
```

Comment:
major: we need to introduce some structure to the list. The way to do this is to use the [expandable sections](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab). 

We can start with the first section "Experiments table"